Exiled.Events/EventArgs/Scp939/ChangingFocusEventArgs.cs

```diff
@@ -0,0 +1,52 @@
+// -----------------------------------------------------------------------
+// <copyright file="ChangingFocusEventArgs.cs" company="Exiled Team">
+// Copyright (c) Exiled Team. All rights reserved.
+// Licensed under the CC BY-SA 3.0 license.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Exiled.Events.EventArgs.Scp939
+{
+    using API.Features;
+    using Interfaces;
+
+    /// <summary>
+    ///     Contains all information before SCP-939 changes its target focus.
+    /// </summary>
+    public class ChangingFocusEventArgs : IPlayerEvent, IDeniableEvent
+    {
+        /// <summary>
+        ///     Initializes a new instance of the <see cref="ChangingFocusEventArgs" /> class.
+        /// </summary>
+        /// <param name="player">
+        ///     <inheritdoc cref="Player" />
+        /// </param>
+        /// <param name="state">
+        ///     The state of the focus.
+        /// </param>
+        /// <param name="isAllowed">
+        ///     <inheritdoc cref="IsAllowed" />
+        /// </param>
+        public ChangingFocusEventArgs(Player player, bool state, bool isAllowed = true)
+        {
+            Player = player;
```

Comment:
We should get the player in every event constructor from its `ReferenceHub` like you do in every other event below.